### PR TITLE
Introduces Submodules

### PR DESCRIPTION
**Pull request by KaiVolland**
This introduces gitsubmodules:
- client
- login
- lib/BasiGX
- lib/geoext3

If we merge this the readme has to be adapted.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[submodule "src/main/webapp/client"]
+	path = src/main/webapp/client
+	url = https://github.com/terrestris/momo3-frontend
+[submodule "src/main/webapp/lib/BasiGX"]
+	path = src/main/webapp/lib/BasiGX
+	url = https://github.com/terrestris/BasiGX
+[submodule "src/main/webapp/lib/geoext3"]
+	path = src/main/webapp/lib/geoext3
+	url = https://github.com/geoext/geoext3
```
Comment (annarieger) [inline .gitmodules:9]: `BasiGX` and `GeoExt` submodules will be already used in frontend  --> [[.gitmodules]](https://github.com/terrestris/momo3-frontend/blob/master/.gitmodules)

Maybe we can reuse them?

Note:
if not, it probably would be better to adapt the submodules names at least:
frontend `basigx` and `geoext` against `BasiGX` and `geoext3` in backend

**Comment by marcjansen** [comment]
Yeah, thanks. Please merge and then update the readme, @kaivolland

File: .gitmodules (hunk 1 above)
Comment (marcjansen) [inline .gitmodules:9]: I think we don't need them anymore. @kaivolland?

**Comment by annarieger** [comment]
It would be great, if you would consider my remark regarding dublicated modules with the same content and different module names..

Otherwise +1

File: .gitmodules (hunk 1 above)
Comment (KaiVolland) [inline .gitmodules:9]: I allready changed this in the frontend. The frontend now shares BasiGX and GeoExt with our other ext apps.

https://github.com/terrestris/momo3-frontend/commit/749b38e8e95f484404c0cc62bec031818030920e

PS: From know on i will work with PullRequests. :innocent:

**Pull request merged**